examples/s3-sdk-extension-examples/src/test/scala/za/co/absa/atum/utils/SparkJobRunnerMethods.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.utils
+
+import org.scalatest.funsuite.AnyFunSuiteLike
+import scala.language.reflectiveCalls
+
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe
+
+trait SparkJobRunnerMethods {
```

Comment:
I see this is in both examples and s3-examples. 1to1 copy. Is that needed?

Reply:
Yes, it is 1:1 copy. Well, both projects need it to run tests and they are not depending on each other. What alternative do you have in mind?